pandas/core/indexing.py

```diff
@@ -842,8 +842,12 @@ def _getitem_nested_tuple(self, tup: Tuple):
             if self.name != "loc":
                 # This should never be reached, but lets be explicit about it
                 raise ValueError("Too many indices")
-            with suppress(IndexingError):
-                return self._handle_lowerdim_multi_index_axis0(tup)
+            if self.ndim == 1 or not any(isinstance(x, slice) for x in tup):
```

Comment:
@phofl the more i look at this the weirder it seems to be treating Series/DataFrame differently.  Are you clear on if there's a reason for this vs just historical coincidence?

Reply:
@jbrockmendel 

sorry for the slow reply. I am pretty sure that this is for historical reasons, but would have to take a closer look.

I am currently on vacation until mid July. Can get back to you afterwards, will also have more time for pandas then again.

Reply:
no worries, enjoy your vacation